src/instructlab/model/chat.py

```diff
@@ -789,18 +789,35 @@ def chat_cli(
     sys_prompt = CONTEXTS.get(context, "default")(get_model_arch(pathlib.Path(model)))
     loaded["messages"] = [{"role": "system", "content": sys_prompt}]
 
+    retriever: DocumentStoreRetriever | None = None
+
     # Instantiate retriever if RAG is enabled
     if rag_enabled:
         logger.debug("RAG enabled for chat; initializing retriever")
-        retriever: DocumentStoreRetriever | None = create_document_retriever(
-            document_store_uri=document_store_uri,
-            document_store_collection_name=collection_name,
-            top_k=top_k,
-            embedding_model_path=embedding_model_path,
-        )
+        try:
+            retriever: DocumentStoreRetriever | None = create_document_retriever(
+                document_store_uri=document_store_uri,
+                document_store_collection_name=collection_name,
+                top_k=top_k,
+                embedding_model_path=embedding_model_path,
+            )
+        except FileNotFoundError:
+            logger.error(
+                f"Ingested documents were not found at {document_store_uri}.  Try running `ilab rag convert` or setting the value for `--document-store-uri` in chat to point to an existing output of convert."
+            )
```

Comment:
How do we know that it's `document_store_uri` which couldn't be found and not `embedding_model_path`? I recommend catching the error and printing information from it here:

```suggestion
        except FileNotFoundError as e: 
            logger.error(
                f"Failed to create retriever: {e}\n"
                "Please make sure all paths are correct and exist on disk."
            )
```

I really like the error message you gave when you logged the error, so I would rephrase the last line of my suggestion into one that's more useful like what you have.